Xamarin.Forms.Platform.Android/AppCompat/ImageButtonRenderer.cs

```diff
@@ -42,7 +42,7 @@ public class ImageButtonRenderer :
 		ViewGroup IVisualElementRenderer.ViewGroup => null;
 		VisualElementTracker IVisualElementRenderer.Tracker => _tracker;
 
-		ImageButton ImageButton
+		public ImageButton ImageButton
 		{
 			get => _imageButton;
 			set
```

Comment:
Make the set private and then I'd say rename the property to Element so that it's consistent with how other renderers expose the name of the Forms Control.

Reply:
ImageButton is renamed to Element with a private set.